docs/production-deployment/self-hosted-guide/checklist.mdx

```diff
@@ -46,106 +77,81 @@ At a high level, you will want to track these 3 categories of metrics:
-Leave your old Workers running, possibly reducing the number of instances, until no more open Workflows are on `tq-v1`.
-Then you can decommission all of them.
+Temporal service availability can be affected by server upgrades.
+We recommend load and availability testing during the upgrade process to understand the performance envelope.
 
-**Advantages**
+Temporal Cloud updates are managed by the Temporal Cloud team.
+Cloud upgrades are seamless.
 
-- Conceptually simple.
-- Robust.
-  Changes are isolated from each other in a way that makes mistakes unlikely.
+## Expert Support
 
-**Disadvantages**
+Temporal recommends that customer platform teams who are building out a Temporal service gain deep experience across the lifecycle and breadth of a Temporal application.
 
-- Operationally complex (need to keep old Workers alive and change the Task Queue that clients point to).
-  This approach can result in multiple sets of old Workers, especially with long-running Workflows.
-- Can't be used to fix a bug in currently running or open Workflows.
+Specific activities include:
+- [Worker tuning](/develop/worker-performance)
+- [Worker best practices](/workers)
+- Code reviews
+- Design guidance
+- Training
+- Code reviews
+- Security reviews
+- [Metrics](/references/sdk-metrics) and monitoring
+- Technical onboarding
 
-### Patch and GetVersion APIs
+Temporal support provides guidance on all of the above.
+Temporal support will work with your teams to make this easier and support teams that support Temporal.
 
-By using functions available in our SDKs, you can branch in your Workflow code based on whether the Workflow is running with newer or older code.
-The APIs take slightly different forms depending on the language you're using.
-For details, see the language-specific links in [Workflow Versioning](/workflows#workflow-versioning).
 
-**Advantages**
+## Cost Management
 
-- Don't need to change where Workflow starters are pointing.
-- Lets you change the yet-to-be-executed behavior of currently open Workflows while remaining compatible with existing Event Histories.
-  The behavior of new Workflows always takes the "new" path.
+Running a mission critical, global Temporal Service can be expensive.
+Temporal Server is a complex system to run and scale.
+We recommend performance testing as outlined above, and planning scaling around your performance requirements.
+This implies that Temporal Servers will often be over-sized, but this is necessary to handle unpredictable spiky workloads.
+Performance testing can help you right-size your environments.
+Running mission critical Temporal as a Service requires multiple Temporal Clusters for high availability and global coverage. 
 
-**Disadvantages**
+It is a good practice to have trained, experienced administrators familiar with Temporal server architecture to maintain Temporal servers, with global coverage to maintain a global, mission critical service.
```

Comment:
Active: to train seasoned administrators familiar with. Should Server be capped here? Maybe? 
Maintain _your_ self-hosted Temporal servers and use the global (capped?) coverage features to maintain a ...

Love the emphasis on mission-critical service.

Reply:
replaced the first mention of server with Service, I think that helped:

"It is a good practice to have trained, experienced administrators familiar with Temporal Service architecture to maintain your Temporal servers and provide a mission critical service."